nilearn/plotting/tests/test_displays.py

```diff
@@ -38,3 +41,26 @@ def test_demo_ortho_projector():
     with tempfile.TemporaryFile() as fp:
         oprojector.savefig(fp)
     oprojector.close()
+
+
+# Smoke test to check threshold function for plot connectome
+def test_threshold_plot_connectome():
+    adjacency_matrix = np.array([[1., 2.],
+                                 [2., 1.]])
+    name = 'edge_threshold'
+    calculate = 'fast_abs_percentile'
+    # a few not correctly formatted strings for 'edge_threshold'
+    wrong_edge_thresholds = ['0.1', '10', '10.2.3%', 'asdf%']
+    for wrong_edge_threshold in wrong_edge_thresholds:
+        assert_raises_regex(ValueError,
+                            'should be a number followed by the percent sign',
```

Comment:
While you are at it, you should check that `name` is used in the error message. You can use a regexp like:
'{0}.+should be a number followed by the percent sign'.format(name)`